### DISCUSSION TITLE
Mongodb connection issue

### DISCUSSION THREAD

**Discussion by surajgupt01** [Note-Vault/Note-Vault#297] [General]
I am not able to connect to mongodb can anyone help me with that

**Comment by VaibhavArora314** [comment]
Can you provide more context to what issue you are facing?

**Reply by surajgupt01** [reply]
in the contribution guideline it is shown to connect to the port on Mongodb compass (27017) I have done it but its is still showing error is there any other way to do it
![Screenshot 2024-05-26 220247](https://github.com/Note-Vault/Note-Vault/assets/94984348/0872a381-1e2f-450e-9d01-66b490bbfe47)
![Screenshot 2024-05-26 220633](https://github.com/Note-Vault/Note-Vault/assets/94984348/c9b7b2a9-9f01-46e5-a09b-393aad13c67a)

**Reply by VaibhavArora314** [reply]
Did u create a .env file?

**Reply by surajgupt01** [reply]
![Screenshot 2024-05-26 223148](https://github.com/Note-Vault/Note-Vault/assets/94984348/93b18fa6-80a1-41a3-bf82-99f07fed9862)
![Screenshot 2024-05-26 223155](https://github.com/Note-Vault/Note-Vault/assets/94984348/5939059d-5d08-40a3-af62-4d7bceea903d)

**Reply by VaibhavArora314** [reply]
The name of the env file should be .env instead of config.env
